test/test_facevalues.jl

```diff
@@ -13,70 +13,119 @@ for (scalar_interpol, quad_rule) in (
+                    @test function_divergence(fv, i, ue) ≈ tr(Gqp)
+                    rdim == 3 && @test function_curl(fv, i, ue) ≈ Ferrite.curl_from_gradient(Gqp)
+                else
+                    @test function_divergence(fv, i, ue) ≈ sum(Gqp)
+                end
+            end
+
+            #Test CellValues when input is a ::Vector{<:Vec} (most of which is deprecated)
+            ue_vec = [zero(Vec{rdim,Float64}) for i in 1:n_basefunc_base]
+            G_vector = rand(Tensor{2, rdim})
             for i in 1:n_basefunc_base
-                u[i] = H ⋅ xs[i]
-                u_scal[i] = V ⋅ xs[i]
+                ue_vec[i] = G_vector ⋅ coords[i]
             end
-            u_vector = reinterpret(Float64, u)
+
             for i in 1:getnquadpoints(fv)
-                @test getnormal(fv, i) ≈ n[face]
                 if func_interpol isa Ferrite.ScalarInterpolation
-                    @test function_gradient(fv, i, u) ≈ H
-                    @test function_symmetric_gradient(fv, i, u) ≈ 0.5(H + H')
-                    @test function_divergence(fv, i, u_scal) ≈ sum(V)
-                    @test function_divergence(fv, i, u) ≈ tr(H)
-                    @test function_gradient(fv, i, u_scal) ≈ V
-                    rdim == 3 && @test function_curl(fv, i, u) ≈ Ferrite.curl_from_gradient(H)
-                    function_value(fv, i, u)
-                    function_value(fv, i, u_scal)
-                else # func_interpol isa Ferrite.VectorInterpolation
-                    @test function_gradient(fv, i, u_vector) ≈ H
-                    @test (@test_deprecated function_gradient(fv, i, u)) ≈ H
-                    @test function_symmetric_gradient(fv, i, u_vector) ≈ 0.5(H + H')
-                    @test (@test_deprecated function_symmetric_gradient(fv, i, u)) ≈ 0.5(H + H')
-                    @test function_divergence(fv, i, u_vector) ≈ tr(H)
-                    @test (@test_deprecated function_divergence(fv, i, u)) ≈ tr(H)
+                    @test function_gradient(fv, i, ue_vec) ≈ G_vector
+                else# func_interpol isa Ferrite.VectorInterpolation
+                    @test (@test_deprecated function_gradient(fv, i, ue_vec)) ≈ G_vector
+                    @test (@test_deprecated function_symmetric_gradient(fv, i, ue_vec)) ≈ 0.5(G_vector + G_vector')
+                    @test (@test_deprecated function_divergence(fv, i, ue_vec)) ≈ tr(G_vector)
                     if rdim == 3
-                        @test function_curl(fv, i, u_vector) ≈ Ferrite.curl_from_gradient(H)
-                        @test (@test_deprecated function_curl(fv, i, u)) ≈ Ferrite.curl_from_gradient(H)
+                        @test (@test_deprecated function_curl(fv, i, ue_vec)) ≈ Ferrite.curl_from_gradient(G_vector)
                     end
-                    @test function_value(fv, i, u_vector) ≈ (@test_deprecated function_value(fv, i, u))
+                    function_value(fv, i, ue_vec)
```

Comment:
This gives a warning because it is deprecated, why is this teste removed and there is just a function call?